plugins/versionpress/src/Git/CommitMessage.php

```diff
@@ -57,6 +57,19 @@ public function getSubject()
     }
 
     /**
+     * Returns commit message subject without prefix set in constant VP_COMMIT_MESSAGE_PREFIX
+     * 
+     * @return string
+     */
+    public function getUnprefixedSubject()
```

Comment:
From my point of view it is same service of `CommitMessage` class as its method `getVersionPressTags`. Tags are also not added as constructor args to `CommitMessage` instance but the instance itself returns them.